test/Microsoft.AspNetCore.ResponseCaching.Tests/TestUtils.cs

```diff
@@ -30,18 +31,29 @@ static TestUtils()
             StreamUtilities.BodySegmentSize = 10;
         }
 
-        internal static RequestDelegate TestRequestDelegate = async (context) =>
+        internal static RequestDelegate TestRequestDelegate = async context =>
         {
-            var uniqueId = Guid.NewGuid().ToString();
             var headers = context.Response.GetTypedHeaders();
-            headers.CacheControl = new CacheControlHeaderValue()
+
+            string expires = context.Request.Query["Expires"];
```

Comment:
Then it would be `StringValues`. `string` enforces the implicit conversion to happen.